pages/theming/css-framework.html

```diff
@@ -111,9 +111,9 @@
 <h4>Overlay &amp; Shadow</h4>
 <ul>
 	<li>
-		<code>.ui-widget-overlay</code>: Applies 100% width &amp; height dimensions to an overlay screen, along with background color/texture, and screen opacity.
```

Comment:
Isn't that still theme dependent? If you pick or create a theme with an overlay texture, you still get a texture. Assuming we're putting those variables back.